### PR TITLE
PHPCS: php/datasource/class-fieldmanager-datasource-post.php

### PR DESCRIPTION
**Pull request by kjbenk**
# Issue
#271 

# Description
PHPCS for file:

- `php/datasource/class-fieldmanager-datasource-post.php`

### CODE REVIEW

File: php/datasource/class-fieldmanager-datasource-post.php [hunk 1]
```diff
@@ -7,429 +12,478 @@
...
+	 *
+	 * @var boolean
+	 */
+	public $use_ajax = true;
+
+	/**
+	 * If not empty, set this post's ID as a value on the linked post. This is used to
+	 * establish two-way relationships.
+	 *
+	 * @var string|Null
```
Comment (davisshaver) [inline php/datasource/class-fieldmanager-datasource-post.php]: s/`Null`/`null`

File: php/datasource/class-fieldmanager-datasource-post.php [hunk 2]
```diff
@@ -7,429 +12,478 @@
...
+		return $value;
+	}
+
+	/**
+	 * Handle any actions based on the parent's status transition.
+	 *
+	 * @param Fieldmanager_Field $field The parent.
+	 * @param int                $value The child post id.
+	 */
+	public function presave_status_transition( Fieldmanager_Field $field, $value ) {
```
Comment (davisshaver) [inline php/datasource/class-fieldmanager-datasource-post.php]: Typo with `Fieldmanager_Field`?

File: php/datasource/class-fieldmanager-datasource-post.php [hunk 3]
```diff
@@ -7,429 +12,478 @@
...
+	/**
+	 * Preload alter values for post parent
+	 * The post datasource can store data outside FM's array.
+	 * This is how we add it back into the array for editing.
+	 *
+	 * @param  Fieldmanager_Field $field  The field.
+	 * @param  array              $values The loaded values.
+	 * @return array              $values Loaded up, if applicable.
+	 */
+	public function preload_alter_values( Fieldmanager_Field $field, $values ) {
```
Comment (davisshaver) [inline php/datasource/class-fieldmanager-datasource-post.php]: I think this is a typo too

File: php/datasource/class-fieldmanager-datasource-post.php (hunk 2 above)
Comment (kjbenk) [inline php/datasource/class-fieldmanager-datasource-post.php]: No it seems correct to me.

File: php/datasource/class-fieldmanager-datasource-post.php (hunk 3 above)
Comment (kjbenk) [inline php/datasource/class-fieldmanager-datasource-post.php]: `Fieldmanager_Field` seems correct to me as well here.

**Pull request merged**